pkg/common/fault/constants.go

```diff
@@ -23,7 +23,8 @@ const (
 	CSIVmUuidNotFoundFault = "csi.fault.nonstorage.VmUuidNotFound"
 	// CSIVmNotFoundFault is the fault type when VM object is not found in the VC
 	CSIVmNotFoundFault = "csi.fault.nonstorage.VmNotFound"
-
+	// CSIDiskNotDetachedFault is the fault type when disk is still attached to the vm
+	CSIDiskNotDetachedFault = "csi.fault.nonstorage.DiskNotDetachedFault"
```

Comment:
nit: csi.fault.nonstorage.DiskNotDetachedFault -> csi.fault.nonstorage.DiskNotDetached, keeping it consistent with other faults.